frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py

```diff
@@ -35,14 +35,18 @@ def send_scheduled_message(self) -> dict:
             "WhatsApp Templates", self.template,
             fieldname='language_code'
         )
+        template_actual_name = frappe.db.get_value(
```

Comment:
instead of making a database call. We can save name field in whatsapp template  as self.template_name.lower().replace(" ", "_")